tsg/config.py

```diff
@@ -21,6 +21,7 @@
 os.makedirs(INTERMEDIATE_DIR, exist_ok=True)
 
 FIELDS = ['type', 'title', 'isbn', 'content']
+FIELD_WEIGHTS = [2.5, 2.5, 5, 1]
```

Comment:
If you believe so i think it's ok. When evaluating quality of search results, this could be a factor. Although i think is good to go for merging.